src/core/lightning/index.ts

```diff
@@ -102,31 +83,18 @@ export const LightningMixin = (superclass) =>
         throw new LndOfflineError("no active lnd to create an invoice")
       }
 
-      try {
-        input = {
-          lnd,
-          tokens: value,
-          expires_at,
-        }
+      const lndService = MakeLndService(lnd)
```

Comment:
if we will use this change then lightning service must manage the lnd nodes (this layer must not handle auth lnds initialization)

Reply:
yes lnd specific auth logic needs to migrate to the lnd service implementation - agreed